src/toolkits/drawing_classifier/dc_data_iterator.hpp

```diff
@@ -67,6 +67,9 @@ class data_iterator {
 
     /** Determines results of shuffle operations if enabled. */
     int random_seed = 0;
+
+    // normalization factor for input data
+    float norm_fator = 255.0f;
```

Comment:
`norm_factor`?

Reply:
scale (normalization) factor for each pixel value. More acurately, it should be pixel * 1/255.0.

Reply:
renamed to scale_factor in order to avoid ambiguity with norm distribution.